Makefile

```diff
@@ -151,8 +155,24 @@ dbuild: crioimage
 integration: crioimage
 	$(CONTAINER_RUNTIME) run -e STORAGE_OPTIONS="--storage-driver=vfs" -e TEST_USERNS -e TESTFLAGS -e TRAVIS -t --privileged --rm -v ${CURDIR}:/go/src/${PROJECT} ${CRIO_IMAGE} make localintegration
 
-testunit:
-	$(GO) test -tags "$(BUILDTAGS) containers_image_ostree_stub" -cover $(PACKAGES)
```

Comment:
Will those unit tests still be executed?

Reply:
Yes, absolutely. Ginkgo runs all existing test suites as usual, but not with these fancy dot for every single spec:

```fish
/home/sascha/go/src/github.com/kubernetes-sigs/cri-o/build/bin/ginkgo \
         \
        -r \
        --cover \
        --covermode atomic \
        --outputdir /home/sascha/go/src/github.com/kubernetes-sigs/cri-o/build/coverage \
        --coverprofile coverprofile \
        --tags "containers_image_ostree_stub      seccomp selinux " \
        --succinct
=== RUN   TestConfigToFile
--- PASS: TestConfigToFile (0.00s)
=== RUN   TestConfigUpdateFromFile
--- PASS: TestConfigUpdateFromFile (0.00s)
PASS
coverage: 4.1% of statements
=== RUN   TestFindEngine
--- PASS: TestFindEngine (1.00s)
PASS
coverage: 85.7% of statements
[1545226481] Registrar - 14/14 specs •••••••••••••• SUCCESS! 1.17814ms PASS
coverage: 97.8% of statements
=== RUN   TestResolveNames
--- PASS: TestResolveNames (0.00s)
PASS
coverage: 4.9% of statements
=== RUN   TestUpdateFromFile
--- PASS: TestUpdateFromFile (0.00s)
=== RUN   TestToFile
--- PASS: TestToFile (0.00s)
=== RUN   TestAddOCIBindsForDev
--- PASS: TestAddOCIBindsForDev (0.00s)
=== RUN   TestGetInfo
--- PASS: TestGetInfo (0.00s)
=== RUN   TestGetContainerInfo
--- PASS: TestGetContainerInfo (0.00s)
=== RUN   TestGetContainerInfoCtrNotFound
--- PASS: TestGetContainerInfoCtrNotFound (0.00s)
=== RUN   TestGetContainerInfoCtrStateNil
--- PASS: TestGetContainerInfoCtrStateNil (0.00s)
=== RUN   TestGetContainerInfoSandboxNotFound
--- PASS: TestGetContainerInfoSandboxNotFound (0.00s)
=== RUN   TestParseDNSOptions
--- PASS: TestParseDNSOptions (0.00s)
=== RUN   TestMergeEnvs
--- PASS: TestMergeEnvs (0.00s)
PASS
coverage: 5.3% of statements
path is /home/sascha/go/src/github.com/kubernetes-sigs/cri-o/build/coverage
All profiles combined

Ginkgo ran 5 suites in 7.205249807s
Test Suite Passed
```